README.md

```diff
@@ -5,8 +5,8 @@ implemented in Go. It is used to sign transactions for use
 on [Vega](#about-vega). Vega Wallet creates and manages HD wallets with ed25519
 key pairs.
 
-### Use the [getting started instructions](/GETTING_STARTED.md) to install and run the wallet if you are new to command line (CLI).
-
+### ⚠️ [Learn how to install and run Vega Wallet (v 0.8.0) by clicking here.](https://github.com/vegaprotocol/go-wallet/tree/master#vega-wallet) 
```

Comment:
@ValentinTrinque I removed the link to "Getting Started" here because the instructions are for novices, and so the likelihood of them working from develop is almost 0. Once the new version is in a release, I'll put it back.